documentation/general/dash-sonic-hld.md

```diff
@@ -305,14 +305,31 @@ DASH_ACL_GROUP_TABLE:{{group_id}}
     "guid": {{string}}
 ```
 
+```
+DASH_ACL_SRC_TAG_TABLE:{{tag_name}}
```

Comment:
Please rename to DASH_PREFIX_TAG_TABLE. One table is enough for both src and dst as controller does not have to repeat it. Also the same could be used in future for non-ACL cases. So lets not prefix ACL to the table name.